chipsec/helper/win/win32helper.py

```diff
@@ -553,8 +554,9 @@ def native_write_phys_mem( self, phys_address_hi, phys_address_lo, length, buf )
     # @TODO: Temporarily the same as read_phys_mem for compatibility
     def read_mmio_reg( self, phys_address, size ):
         out_size = size
+        logger().log("size: {} addr: {}".format(size, phys_address))
```

Comment:
Should this always log?

Reply:
Good catch. I thought I fixed all of the logging.